CMakeLists.txt

```diff
@@ -77,6 +77,17 @@ if (OPAE_MINIMAL_BUILD)
     set(CMAKE_INSTALL_PREFIX "/usr" CACHE STRING "" FORCE)
 endif(OPAE_MINIMAL_BUILD)
 
+############################################################################
+## CMAKE_INSTALL_PREFIX check ##############################################
+############################################################################
+
+string(LENGTH ${CMAKE_INSTALL_PREFIX} INSTALL_PREFIX_LEN)
+if (${INSTALL_PREFIX_LEN} GREATER 127)
```

Comment:
Should this be an error instead of just a message? We already know the commands will fail mysteriously. I'm worried that just a message will be lost in the noise of everything else CMake is spitting out.